src/ParquetMsg.chpl

```diff
@@ -1709,15 +1712,15 @@ module ParquetMsg {
     }
         
     try {
-      extern proc c_getDatasetNames(filename, dsetResult, errMsg): int(32);
+      extern proc c_getDatasetNames(filename, dsetResult, readNested, errMsg): int(32);
```

Comment:
There is one other usage in this file (lines 914 and 922) that didn't get updated to use this new function. I've opened https://github.com/Bears-R-Us/arkouda/pull/2404 to fix this.